localenv/mock-account-servicing-entity/Dockerfile

```diff
@@ -7,10 +7,6 @@ ENV PATH="$PNPM_HOME:$PATH"
 
 RUN corepack enable
 RUN corepack prepare pnpm@8.7.4 --activate
-RUN apk add --no-cache \
-    python3 \
-    make \
-    g++
```

Comment:
was breaking on windows, and looks like not needed in general

Reply:
@raducristianpopa specified in the original pr that this is for `node-gyp`. https://github.com/interledger/rafiki/pull/1795#discussion_r1307418635 Could `node-gyp` be included with pnpm? It's liable to be used by some of our packages and its hard to say what. I did remove from backend and start localenv OK FWIW. This SO post suggests its part of npm at least https://stackoverflow.com/questions/67584602/does-node-gyp-come-with-npm

Reply:
There was a dependency chain within `testcontainers` that needed these packages (Pything and the C++ compiler) and I was following [their requirements](https://github.com/nodejs/node-gyp?tab=readme-ov-file#on-unix). I recall discussing with Max about this some time ago and it was working for me without installing them at that time.